cartridge/utils.lua

```diff
@@ -356,13 +356,21 @@ local function http_read_body(req)
     -- that have a special meaning with % to escape them.
```

Comment:
Could you please add a function description in ldoc style if you have time?

Reply:
Added